components/automate-ui/src/app/pages/+compliance/+profile/+profile-overview/profile-overview.component.html

```diff
@@ -199,7 +199,7 @@ <h2 id="upload-modal-title" slot="title">Upload an archived profile (.tar.gz or
                   </a>
                 </chef-table-cell>
                 <chef-table-cell>{{ profile.version }}</chef-table-cell>
-                <chef-table-cell>
+                <chef-table-cell class="text-right">
```

Comment:
Slight concern over why this table is unique...?

Reply:
in order to make the ellipses apply, content needs to be `inline-block`, which then caused - in this elements case - it to no longer be right aligned.  A simple and low code fix is to use `text-align: right` which I wanted to be reusable incase it needed to be used elsewhere.
The reason it was happening only here (that I know of yet) is that most of the columns contain text, where as this column contain an icon so it was thrown off in a way the others weren't